.github/workflows/ci-pull-requests.yml

```diff
@@ -70,7 +70,5 @@ jobs:
       - name: Fetch base branch for `git diff`
         run: git fetch origin ${{ github.event.pull_request.base.ref }}:${{ github.event.pull_request.base.ref }}
 
-      - name: Run ESLint on changes only
-        env:
-          ESLINT_PLUGIN_DIFF_COMMIT: ${{ github.event.pull_request.base.ref }}
+      - name: Lint
```

Comment:
According to [the original PR](https://github.com/n8n-io/n8n/pull/4137/files#diff-4eb787923b5d10508f08732ecbbe5195a389acb35bb1d56c7b57ccaa747d9c7c) we also need to remove the prior `Fetch base branch` step.